pkg/cloudprovider/plugins/azure/azure_resourcegraph.go

```diff
@@ -42,10 +43,16 @@ func (p *azureServiceSdkConfigProvider) resourceGraph() (azureResourceGraphWrapp
 }
 
 func invokeResourceGraphQuery(resourceGraphAPIClient azureResourceGraphWrapper, query *string,
-	subscriptions []*string) (interface{}, int64, error) {
+	subscriptions []*string) ([]interface{}, int64, error) {
+	var data = make([]interface{}, 0)
```

Comment:
is this data slice of slice.. meaning [page1]....Vms [page2].... ???

Reply:
Data returned by the API is expected to be an array(type is interface{} though as per query response), I am just appending this data in a slice and returning to the caller. Caller decodes each page.  Any comments here?